cinder/manifest.jsonnet

```diff
@@ -0,0 +1,193 @@
+local kpm = import "kpm.libjsonnet";
+
+function(
+  params={}
+)
+
+kpm.package({
+  package: {
+    name: "stackanetes/cinder",
+    expander: "jinja2",
+    author: "Quentin Machu",
+    version: "0.1.0",
+    description: "cinder",
+    license: "Apache 2.0",
+  },
+
+  variables: {
+    deployment: {
+      node_label: "openstack-control-plane",
+      replicas: 1,
+
+      image: {
+        base: "quay.io/stackanetes/stackanetes-%s:barcelona",
+
+        init: $.variables.deployment.image.base % "kolla-toolbox",
+        db_sync: $.variables.deployment.image.base % "cinder-api",
+        api: $.variables.deployment.image.base % "cinder-api",
+        volume: $.variables.deployment.image.base % "cinder-volume",
+        scheduler: $.variables.deployment.image.base % "cinder-scheduler",
+      },
+    },
+
+    network: {
+      ip_address: "{{ .IP }}",
+      port: 8776,
+
+      ingress: {
+        enabled: true,
+        host: "%s.openstack.cluster",
+        port: 30080,
+
+        named_host: $.variables.network.ingress.host % "volume",
+      },
+    },
+
+    database: {
+      address: "mariadb",
+      port: 3306,
+      root_user: "root",
+      root_password: "password",
```

Comment:
`manifest.jsonnet` is the configuration file, the values here are the default that can be changed via "stackanetes/parameters.yaml' or via the command line.